test/Sentry.Tests/AppDomainUnhandledExceptionIntegrationTests.cs

```diff
@@ -56,6 +59,27 @@ public void Handle_TerminatingTrue_DisposesHub()
             disposableHub.Received(1).Dispose();
         }
 
+        [Fact]
+        public void Handle_TerminatingTrue_IsHandled()
+        {
+            var sut = _fixture.GetSut();
+            sut.Register(_fixture.Hub, SentryOptions);
+
+            var exception = new Exception();
+            sut.Handle(this, new UnhandledExceptionEventArgs(exception, true));
+            Assert.Equal(exception.Data["Sentry:Handled"] as bool?, (bool?)false);
```

Comment:
The order here's inverted. Expected is on the left. But could just be:
```suggestion
            Assert.False((bool)exception.Data["Sentry:Handled"]);
```

Also now could use the `static readonly reference` from `Mechanism` instead of the string here